ha/core/event_analyzer/parser/parser.py

```diff
@@ -145,3 +152,60 @@ def parse_event(self, msg: str) -> HealthEvent:
 
         except Exception as e:
             raise EventParserException(f"Failed to parse IEM. Message: {msg}, Error: {e}")
+
+
+class ClusterResourceParser(Parser):
+    """
+    Subscriber for event analyzer to pass msg.
+    """
+
+    def __init__(self):
+        """
+        Init method.
+        """
+        super(ClusterResourceParser, self).__init__()
+        ConfigManager.init("event_analyzer")
+        Log.info("ClusterResource Parser is initialized ...")
+
+    def parse_event(self, msg: str) -> HealthEvent:
+        """
+        Parse event.
+        Args:
+            msg (str): Msg
+        """
+        try:
+            message = json.dumps(ast.literal_eval(msg))
+            cluster_resource_alert = json.loads(message)
+            cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
+            site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
+            rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+            timestamp = str(int(time.time()))
```

Comment:
This can be deleted

Reply:
We can not remove this. Its needed for event_id creation